test/arrays.js

```diff
@@ -1,6 +1,7 @@
 (function() {
+  var _ = typeof require == 'function' ? require('../underscore') : window._;
```

Comment:
You can just use `require("../")` so if we change the main file name in `package.json`, we don't have to change it everywhere.